HW2/05-form.html

```diff
@@ -41,7 +41,7 @@ <h1 class="text-center">Exercise 05 - Form</h1>
           <label class="form-check-label" for="exampleCheck3">Full Stack Web Dev<br>
```

Comment:
There are three WAVE audit errors and 4 alerts on this page when I run it in regards to your courses taken section. looks like, for each pair of label and input tags the label needs the "for" attribute set to the same exact value of the input "id" attribute value.